mod/openid.php

```diff
@@ -18,17 +18,23 @@ function openid_content(&$a) {
 
 		if($openid->validate()) {
 
-			$authid = normalise_openid($_REQUEST['openid_identity']);
+			#$authid = normalise_openid($_REQUEST['openid_identity']);
+			$authid = $_REQUEST['openid_identity'];
 
 			if(! strlen($authid)) {
 				logger( t('OpenID protocol error. No ID returned.') . EOL);
 				goaway(z_root());
 			}
 
+			// NOTE: we search both for normalised and non-normalised form of $authid
+			//       because the normalization step was removed from setting
+			//       mod/settings.php in 8367cad so it might have left mixed
+			//       records in the user table
+			//
 			$r = q("SELECT `user`.*, `user`.`pubkey` as `upubkey`, `user`.`prvkey` as `uprvkey` 
-				FROM `user` WHERE `openid` = '%s' AND `blocked` = 0 
+				FROM `user` WHERE ( openid = '%s' OR openid = '%s' ) AND blocked = 0
```

Comment:
I've pushed a commit to quote all fields, and removed the aliases
after checking no code path was using them.